cluster/src/main/scala/org/apache/spark/ui/SnappyDashboardPage.scala

```diff
@@ -275,9 +275,22 @@ private[ui] class SnappyDashboardPage (parent: SnappyDashboardTab)
     clusterStatsMap += ("totalOffHeapUsage" -> totalOffHeapUsage)
     clusterStatsMap += ("jvmHeapUsage" -> jvmHeapUsage)
     clusterStatsMap += ("totalJvmHeapUsage" -> totalJvmHeapUsage)
+    clusterStatsMap += ("totalExecutorCoresCount" -> getTotalExecutorCoresCount)
 
   }
 
+  private def getTotalExecutorCoresCount(): Int = {
+    var totalNumExecutorCores = 0
+    val executorToTaskSummary = parent.parent.executorsListener.executorToTaskSummary
+    val executorKeysItr = executorToTaskSummary.keySet.iterator
+    while(executorKeysItr.hasNext){
+      val execKey = executorKeysItr.next()
+      val executor = executorToTaskSummary.get(execKey)
+      totalNumExecutorCores += executor.get.totalCores
+    }
+    totalNumExecutorCores
+  }
+
```

Comment:
You could simply use SnappyContext.totalCoreCount.get()

Reply:
Thanks Sumedh for suggesting that. I have made the changes now.